### ISSUE TITLE
integer overflow in looping average

### ISSUE THREAD

**Issue by Kiskae** [schweikert/fping#195]
When in loop mode the `avg` result will very rapidly turn into garbage data because the `h->total_time` value overflows into negative.

This appears to have always happened, but because of the recent switch from measuring in milliseconds to microseconds this happens much quicker when the measurements happen over wifi or if the ping-time is simply very bad. ([commit](https://github.com/schweikert/fping/commit/754a21e2bdf21102e8d98693cef3836d588c144e))

Replacing `int total_time` with `uint64_t total_time` would probably delay the issue, but the value grows too rapidly for the issue to never happen as it is currently implemented.

Test to a remote server with the additional debug code: `printf("total=%d|sent=%d\n", h->total_time, h->num_recv);`

> 168.1.79.229 : [0], 84 bytes, 259 ms (259 avg, 0% loss)
> total=259919100|sent=1
> 168.1.79.229 : [1], 84 bytes, 260 ms (260 avg, 0% loss)
> total=520504700|sent=2
> 168.1.79.229 : [2], 84 bytes, 258 ms (259 avg, 0% loss)
> total=778917200|sent=3
> 168.1.79.229 : [3], 84 bytes, 258 ms (259 avg, 0% loss)
> total=1037166300|sent=4
> 168.1.79.229 : [4], 84 bytes, 258 ms (259 avg, 0% loss)
> total=1295293100|sent=5
> 168.1.79.229 : [5], 84 bytes, 271 ms (261 avg, 0% loss)
> total=1567270300|sent=6
> 168.1.79.229 : [6], 84 bytes, 258 ms (260 avg, 0% loss)
> total=1825492900|sent=7
> 168.1.79.229 : [7], 84 bytes, 258 ms (260 avg, 0% loss)
> total=2083539000|sent=8
> 168.1.79.229 : [8], 84 bytes, 263 ms (-2.2e-07 avg, 0% loss)
> total=-1947458496|sent=9
> 168.1.79.229 : xmt/rcv/%loss = 9/9/0%, min/avg/max = 258/-2.2e-07/271
